src/detect-lua.c

```diff
@@ -1243,87 +1242,48 @@ static int LuaMatchTest01(void)
     StreamTcpInitConfig(true);
 
     DetectEngineCtx *de_ctx = DetectEngineCtxInit();
-    if (de_ctx == NULL) {
-        goto end;
-    }
+    FAIL_IF_NULL(de_ctx);
     de_ctx->flags |= DE_QUIET;
 
     s = DetectEngineAppendSig(de_ctx, sig);
```

Comment:
When possible, we now prefer to declare the variable in its first usage. So, instead of having `Signature *s = NULL;` on line 1212, you'd declare it here, directly. 

This also applies to Packet variables.